MagneticField/ParametrizedEngine/interface/alpaka/ParabolicParametrizedMagneticField.h

```diff
@@ -0,0 +1,51 @@
+/**
+ Description: Utility function to calculate the Magnetic Field on the GPU
+*/
+
+#ifndef MagneticField_ParametrizedEngine_interface_alpaka_ParabolicParametrizedMagneticField_h
+#define MagneticField_ParametrizedEngine_interface_alpaka_ParabolicParametrizedMagneticField_h
+
+namespace ALPAKA_ACCELERATOR_NAMESPACE {
+
+  namespace MagneticFieldParabolicPortable {
+
+    struct Parameters {
+      // These parameters are the best fit of 3.8T to the OAEParametrizedMagneticField parametrization.
+      // See MagneticField/ParametrizedEngine/src/ParabolicParametrizedMagneticField.cc
+      static constexpr float c1 = 3.8114;
+      static constexpr float b0 = -3.94991e-06;
+      static constexpr float b1 = 7.53701e-06;
+      static constexpr float a = 2.43878e-11;
+      static constexpr float max_radius2 = 13225.f;  // tracker radius
+      static constexpr float max_z = 280.f;          // tracker z
+    };
+
+    template <typename V3>
+    constexpr float Kr(V3 vec) {
```

Comment:
Sorry, I missed this earlier: could you take the `V3` argument by `const &` ?
I would also suggest renaming it to `Vec` or `Vec3`, but that is only a matter of taste:

```suggestion
    template <typename Vec3>
    constexpr float Kr(Vec3 const& vec) {
```

Same for the other functions below.

Reply:
Done: https://github.com/cms-sw/cmssw/pull/44360/commits/89891854e15673587c15a72f8dd29a6528be4fea